### ISSUE TITLE
Add option to disable strict parsing options

### ISSUE THREAD

**Issue by NightMachinary** [ogham/exa#570]
`exa` exits with an error if I use "useless options":

```sh
$ exa -a --oneline -l
Option --oneline (-1) is useless given option --long (-l)
```

This limits my ability to easily create aliases. I want an option where `exa` just chills out. :))

**Comment by NightMachinary** [comment]
Bumping this; I'm starting to think that I should create a wrapper around exa just to solve this problem.

**Comment by NightMachinary** [comment]
```
tldr() { 
	  #nig ea  #not needed because of piping autoremoval of color.
	  isDarwin && { command tldr "$@" || command tldr -p linux "$@" ; return $? }
	  command tldr "$@" | bt
}
exa() {
    local arg long=''
    for arg in "$@"
    do
        [[ "$arg" == "-l" || "$arg" == "--long" ]] && long='y'
    done
    if test -z "$long"
    then
        command exa -1 "$@"
    else
        command exa "$@"
    fi
}
```
This is really awful UX though. It's even a bug considering there is EXA_STRICT in the manpage.

**Comment by sdmtr** [comment]
Another vote for this. I have `ls` aliased to `exa -la` and this issue is pretty annoying.

**Comment by ogham** [comment]
Yeah, this was a bug where strict mode wasn't being checked for. It's been fixed (#476) and will go out in the next release.
